demos/projects/ESPRESSIF/aziotkit/components/azure-iot-kit-sensors/src/oled.c

```diff
@@ -0,0 +1,45 @@
+/* OLED screen Example
+
+   For other examples please check:
+   https://github.com/espressif/esp-iot-solution/tree/master/examples
+
+   This example code is in the Public Domain (or CC0 licensed, at your option.)
+
+   Unless required by applicable law or agreed to in writing, this
+   software is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+   CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+#include <stdio.h>
+#include <time.h>
+#include <sys/time.h>
+#include "oled.h"
+
+void oled_show_signs(ssd1306_handle_t dev);
+
+void oled_init(ssd1306_handle_t dev)
+{
+    oled_clean(dev);
+}
+
+void oled_clean(ssd1306_handle_t dev)
+{
+    iot_ssd1306_clear_screen(dev, 0);
+}
+
+void oled_show_signs(ssd1306_handle_t dev)
+{
+    iot_ssd1306_draw_bitmap(dev, 0, 2, &c_chSingal816[0], 16, 8);

```

Comment:
>16 [](http://example.com/codeflow?start=58&length=2)

Macro for this magic number?

Reply:
all of them so that we can have context associated with number.... like font size